config.yaml

```diff
@@ -22,7 +22,7 @@ listen: :8080
 
     # JWT expiration in seconds ('exp' claim)
     # Defaults to: "604800" (one week)
-# jwt_exp_timeout: 604800
+jwt_exp_timeout: 60
```

Comment:
@alfrunes is this on purpose?

Reply:
Thanks for having my back. No, this was just for verifying that it works (and indeed it does :).

Reply:
Good!